src/lib/mongoBackend/mongoOntimeintervalOperations.cpp

```diff
@@ -85,17 +85,19 @@ HttpStatusCode mongoGetContextSubscriptionInfo
         csiP->attributeList.push_back(attrs[ix].String());
     }
 
-    csiP->expiration = getField(sub, CSUB_EXPIRATION).numberLong();
+    csiP->expiration = sub.hasField(CSUB_EXPIRATION)? getField(sub, CSUB_EXPIRATION).numberLong() : -1;
 
     csiP->url = getStringField(sub, CSUB_REFERENCE);
-    if (sub.hasElement(CSUB_LASTNOTIFICATION)) {
-        csiP->lastNotification = getIntField(sub, CSUB_LASTNOTIFICATION);
+    if (sub.hasElement(CSUB_LASTNOTIFICATION))
+    {
+      csiP->lastNotification = getLongField(sub, CSUB_LASTNOTIFICATION);
```

Comment:
Not sure but... maybe this is other case in which "CSUB_LASTNOTIFICATION might be either Int or Long ..." and we need the "dual read" you have implemented in other places to be safer?